docs/methoddocs/plugins.md

```diff
@@ -69,3 +69,111 @@
     :members:
     :show-inheritance:
 ```
+
+## Multicall
+
+Ape comes with builtin `multicall` method. Yes you're reading this right. No need to install external modules to do multicall, ape got you covered.Perform the Multicall call. This call will trigger again every time the `Call` object is called.
+
+```js
+Raises:
+    :class:`~ape_ethereum.multicall.exceptions.UnsupportedChain`: If there is not an instance of Multicall3 deployed on the current chain at the expected address.
+
+Args:
+    **call_kwargs: the kwargs to pass through to the call handler.
+
+Returns:
+    Iterator[Any]: the sequence of values produced by performing each call stored by this instance.
+```
+
+#### Usage
```

Comment:
In markdown, you need to only increase the number of `#` by one for each new hear.
The parent header has only two `#`s and you are going all the way up to four here.